micrometer-core/src/main/java/io/micrometer/core/instrument/binder/mongodb/MongoMetricsCommandTagsProvider.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2019 VMware, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.core.instrument.binder.mongodb;
+
+import com.mongodb.event.CommandEvent;
+import io.micrometer.core.instrument.Tag;
+
+/**
+ * Provides {@link Tag Tags} for Mongo command metrics.
+ *
+ * @author Chris Bono
+ */
+public interface MongoMetricsCommandTagsProvider {
```

Comment:
What do you think about putting `@FunctionalInterface` on this class?

Reply:
I am ok w/ that and was initially going to but I noticed that it was not used anywhere else in the repo so I left it out for consistency w/ the intention to follow up and ask if it was something not used in MM on purpose or just not used yet. Thanks for the reminder.  

@shakuzen - thoughts?

Reply:
I went ahead and added it as it should not be an issue - if it is I will remove it.